grails-app/services/au/org/ala/merit/AbnLookupService.groovy

```diff
@@ -22,7 +22,8 @@ class AbnLookupService {
     Map lookupOrganisationNameByABN(String organisationABN){
         Map abnDetails
 
-        if (!validateABN(organisationABN)) {
+        organisationABN = validateABN(organisationABN)
+        if (organisationABN == "false") {
```

Comment:
It's better not to return a magic "false" string like this.  Make the call to remove whitespace before calling the validate method and pass the de-whitespaced string to the validate call and you can go back to returning a boolean.